### ISSUE TITLE
I was already used GetMaterialApp, how to use this library with GetX

### ISSUE THREAD

**Issue by zhushenwudi** [harshitverma209/flutter_in_app_pip#1]

**Comment by AsherDubai** [comment]
builder: (context, child) {
              child = EasyLoading.init()(context, child);
              return PiPMaterialApp(
                pipParams: const PiPParams(),
                home: child,
              );
            },

**Comment by harshitverma209** [comment]
Is this resolved?

**Comment by zhushenwudi** [comment]
i will try it later, i am busy in other things

**Comment by zhushenwudi** [comment]
I tried to display a small window on the iOS platform, which is fine in the foreground, but not in the background

**Comment by HarshitVermaTide** [comment]
Yes, this package is for in app pip experience, not background pip.

**Comment by harshitverma209** [comment]
Hello,
This package is for adding PiP inside your own flutter app. Not a background PiP, you can use [android_pip](https://pub.dev/packages/android_pip) for that(android only).
